addons/mail/models/res_users.py

```diff
@@ -41,6 +42,13 @@ def SELF_READABLE_FIELDS(self):
     def SELF_WRITEABLE_FIELDS(self):
         return super().SELF_WRITEABLE_FIELDS + ['notification_type']
 
+    @api.model
+    def _name_search(self, name='', args=None, operator='ilike', limit=100, name_get_uid=None):
```

Comment:
`def _name_search(self, name, domain=None, operator='ilike', limit=None, order=None):`
should be the same with the `super`